release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -201,6 +201,19 @@ Previously, quick start prerequisites were displayed as combined text instead of
 
 image::quick-start-prerequisites.png[quick start prerequisites displayed as popover,338,336]
 
+[id="ocp-4-8-web-console-developer-perspective"]
+==== Developer perspective
+
+* Use custom pipeline templates to create and deploy an application from a Git repository. These templates override the default pipeline templates provided by OpenShift Pipelines 1.5 and later.
+* Filter Helm charts based on their certification level and see all Helm charts in the *Developer Catalog*.
+* Use the options in the *Add* page to create applications and associated services and deploy these applications and services on {product-title}. The *Add* page options include: *Getting started resources*, *Creating applications using samples*, *Build with guided documentation*, and *Explore new developer features*.
+* Use workspaces in pipelines when creating applications in the pipeline builder. Add a trigger to a pipeline so you can use tasks to support workspaces in pipelines.
+* Use the JAR files in the *Topology* view of the *Developer* perspective to deploy your Java applications.
+* Create multiple event source types in {product-title} and connect these source types to sinks. Take functions deployed as a Knative service on a {product-title} cluster and connect them to a sink to Knative Eventing components.
+* Use the `finally` task in the pipeline to execute commands in parallel.
+* Use code assistance in the *Add Task* form to access the task parameter values. To view the pipeline parameter and the correct syntax for referencing that particular pipeline parameter, go to the respective text fields.
+* Run a `Task` only after certain conditions are met. Use the `when`` field to configure your task's executions and to list a series of references to `when` expressions.
```

Comment:
Remove the extra back tick after the first `when` mention.
Avoid using possesives - 's for inanimate things. Suggest ...to configure your task's executions/to configure the executions of your task,
Also suggest adding comma to the word before and.